src/test/java/com/puppycrawl/tools/checkstyle/checks/imports/ImportOrderCheckTest.java

```diff
@@ -818,7 +818,7 @@ public void testClearState() throws Exception {
         assertWithMessage("State is not cleared on beginTree")
                 .that(TestUtil.isStatefulFieldClearedDuringBeginTree(check,
                         staticImport.orElseThrow(), "lastImportStatic",
-                        lastImportStatic -> !((boolean) lastImportStatic)))
```

Comment:
@mahfouz72 @nrmancuso @romani If this parenthesis is unnecessary, then why didn't our check ping it?
https://checkstyle.org/checks/coding/unnecessaryparentheses.html#UnnecessaryParentheses

Reply:
Yes, we need to create issue on this.

Reply:
https://github.com/checkstyle/checkstyle/issues/14872